scripts/pylint_extensions.py

```diff
@@ -32,13 +32,14 @@
 _PYLINT_PATH = os.path.join(_PARENT_DIR, 'oppia_tools', 'pylint-1.9.4')
 sys.path.insert(0, _PYLINT_PATH)
 
+# List of punctuation allowed to used at the end of comments and docstrings.
```

Comment:
punctuation allowed to used --> punctuation symbols that can be used

Reply:
Done.

Reply:
Done.